server/core/src/https/v1.rs

```diff
@@ -17,14 +17,19 @@ use kanidm_proto::internal::{AppLink, IdentifyUserRequest, IdentifyUserResponse}
 use kanidm_proto::v1::{
     AccountUnixExtend, ApiToken, ApiTokenGenerate, AuthIssueSession, AuthRequest, AuthResponse,
     AuthState as ProtoAuthState, CUIntentToken, CURequest, CUSessionToken, CUStatus, CreateRequest,
-    CredentialStatus, DeleteRequest, Entry as ProtoEntry, GroupUnixExtend, ModifyRequest,
+    CredentialStatus, DeleteRequest, Entry as ProtoEntryReal, GroupUnixExtend, ModifyRequest,
     RadiusAuthToken, SearchRequest, SearchResponse, SingleStringRequest, UatStatus, UnixGroupToken,
     UnixUserToken, UserAuthToken, WhoamiResponse,
 };
 use kanidmd_lib::idm::event::AuthResult;
 use kanidmd_lib::idm::AuthState;
 use kanidmd_lib::prelude::*;
 use kanidmd_lib::value::PartialValue;
+// Used by macros
+#[allow(unused_imports)]
+use crate::https::apidocs::PrivateSchema;
+#[allow(unused_imports)]
+use crate::https::apidocs::response_schema::ProtoEntry;
```

Comment:
For some reason this does not get deteceted as used... maybe utoipa isn't actually "using" the types in the macros.

Reply:
As far as I understand it, they're used for inference things at build time but not actually used?